testplan/testing/multitest/base.py

```diff
@@ -95,6 +95,7 @@ def get_options(cls):
             ConfigOption('result', default=Result): is_subclass(Result),
             ConfigOption('thread_pool_size', default=0): int,
             ConfigOption('max_thread_pool_size', default=10): int,
+            ConfigOption('stop_on_exception', default=False): bool,
```

Comment:
Maybe stop_on_error is a more meaningful name.
Also by default we should stop on first exception as always, so the default value should be True.